mantidproject/mantid
# Add option for can subtraction in Indirect Diffraction

Fixes #12976.

This branch has code from #13046, which must be merged first.

To test:
- Open Indirect > Diffraction
- Instrument: OSIRIS, diffonly
  - Run Number: 10203
  - Calibration file: `osiris_041_RES10.cal`
  - Vanadium: 10156
  - Manual dRange: 4
  - Run without can subtraction
  - Plot the dRange workspace and change the Y axis range to -10 to 50
  - Run again with container run: 10241
  - Notice the small background is removed (this is not the ideal data for testing this)
  - Run again changing the container scale factor
- Instrument: IRIS, diffspec
  - Run Number: 27176
  - Run without can subtraction
  - Plot the `IRS26176_diffspec_red` workspace
  - Run again with container run: 26173
  - See that a good amount of the background is removed
  - Run again changing the container scale factor

Release notes updates [here](http://www.mantidproject.org/index.php?title=Release_Notes_3_5_Indirect_Inelastic&diff=24401&oldid=24396).
